analyzer/src/funTest/assets/projects/synthetic/poetry-expected-output.yml

```diff
@@ -0,0 +1,772 @@
+---
+repository:
```

Comment:
The expected output file should be a serialized `ProjectAnalyzerResult`, not a whole `OrtResult` as this contains irrelevant information like the `environment` section below.